src/main/webapp/js/lfbxml/XmlBuilder.js

```diff
@@ -91,6 +92,11 @@
         cType: properties.cType || 'uint8_t',
       };
 
+      if (field_names.indexOf(field.name) !== -1) {
+        throw `Table ${name} need unique field names.`;
```

Comment:
I would prefer something more informative:

```suggestion
        throw `Field ${field.name} is duplicated in table ${name}. Field names must be unique.`;
```